pkgs/top-level/all-packages.nix

```
@@ -24821,6 +24821,8 @@ in
 
   vym = qt5.callPackage ../applications/misc/vym { };
 
+  wad = callPackage ../tools/security/wad { };
```

This package takes other python packages and functions as argument.
It should be called with python3Package's callPackage function:

```suggestion
  wad = python3Packages.callPackage ../tools/security/wad { };
```

---

Changed